### ISSUE TITLE
when i use H_SEGFIX.json to train cityscapes datasets meet the error:

### ISSUE THREAD

**Issue by qingchengboy** [openseg-group/openseg.pytorch#46]
In loss_heleper.py
In the calculation of loss function, the input is two tensors[1,8,128,128] /[1,2,128,128], and the corresponding label of single is three tensors.[1,512,512],[1,512,512],[1,512,512]

targets=targets_.clone().unsqueeze(1).float()
AttributeError:'list' object has no attribute 'clone'

**Comment by PkuRainBow** [comment]
@hsfzxjy Please help to check the possible reasons.

**Comment by hsfzxjy** [comment]
Hi @qingchengboy 

When training SegFix, [SegFixLoss](https://github.com/openseg-group/openseg.pytorch/blob/master/lib/loss/loss_helper.py#L190) should be used. You may post the script you use so that we can figure out the problem.

**Comment by qingchengboy** [comment]
@hsfzxjy Thank you. That's the question. I chose the wrong loss type （”fs_auxce_loss“）for calculation

**Comment by YAwei666** [comment]
@qingchengboy so how to change the script 
of the H_SEGFIX.json?

**Comment by qingchengboy** [comment]
@YAwei666 
In the H_SEGFIX.json
"model_name": "segfix_hrnet"

**Comment by Jonnyboyyyy** [comment]
> 
I am confused.
Is changing 'model_name' enough? 
Do I have to change 'loss_type' too, if I want to train SegFix?
